test/browser/pages/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  RelyingPartyPage: require("./relying-party.js"),
```

Comment:
What is this page? I can't see it in main

Reply:
The index links the page class to a variable that is called in the stepDefinitions, but I've updated the name of the relyingPartyPage class so it matches the variable we're using in stepDefs